sdk/src/main/java/com/ingresse/sdk/services/UserService.kt

```diff
@@ -82,4 +95,63 @@ class UserService(private val client: IngresseClient) {
         val type = object : TypeToken<Response<UserDataJSON>>() {}.type
         mUserDataCall?.enqueue(RetrofitCallback(type, callback))
     }
+
+    /**
+     * Update user basic infos
+     *
+     * @param request - all parameters used for update user infos
+     * @param onSuccess - success callback
+     * @param onError - error callback
+     */
+    fun updateBasicInfos(request: UserBasicInfos, onSuccess: (UserUpdatedDataJSON) -> Unit, onError: (APIError) -> Unit) {
+        mUpdateBasicInfosCall = service.updateBasicInfos(
+            userId = request.userId,
+            apikey = client.key,
+            userToken = request.userToken,
+            params = request
+        )
+
+        val callback = object : IngresseCallback<Response<UserUpdatedJSON>?> {
+            override fun onSuccess(data: Response<UserUpdatedJSON>?) {
+                val response = data?.responseData
+                if (response == null) {
```

Comment:
This could be done in one line
val response = data?.responseData ?: return onError(default)